kolibri/plugins/html5_viewer/options.py

```diff
@@ -0,0 +1,42 @@
+# Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox
+allowable_sandbox_tokens = set(
+    [
+        "allow-downloads-without-user-activation",
+        "allow-forms",
+        "allow-modals",
+        "allow-orientation-lock",
+        "allow-pointer-lock",
+        "allow-popups",
+        "allow-popups-to-escape-sandbox",
+        "allow-presentation",
+        "allow-same-origin",
+        "allow-scripts",
+        "allow-storage-access-by-user-activation ",
+        "allow-top-navigation",
+        "allow-top-navigation-by-user-activation",
+    ]
+)
+
+
+def clean_sandbox(sandbox_string):
+    """
+    Clean up sandbox string to ensure it only contains valid items.
+    """
+    sandbox_tokens = [
+        token
+        for token in sandbox_string.split(" ")
+        if token in allowable_sandbox_tokens
+    ]
+    return " ".join(sandbox_tokens)
```

Comment:
Maybe show a warning if something unrecognized is excluded, to help with catching typos?

Reply:
Addressed!